.azure-pipelines/stage/checks.yml

```diff
@@ -83,6 +83,7 @@ jobs:
       cacheName: $(CI_TARGET)
       envoyBuildFilterExample: $(ENVOY_FILTER_EXAMPLE)
       cacheTestResults: ${{ parameters.cacheTestResults }}
+      diskspaceHack: true
```

Comment:
removes all the default installed and unused dev environments

Reply:
its the nuclear solution - but it will work for now - ill try and make it only happen on coverage in a follow up - but lets get things working (this does)